Editorial review: Document interest invokers (#42021)
mdn/content: files/en-us/web/api/htmlanchorelement/interestforelement/index.md

---
title: "HTMLAnchorElement: interestForElement property"
short-title: interestForElement
slug: Web/API/HTMLAnchorElement/interestForElement
page-type: web-api-instance-property
browser-compat: api.HTMLAnchorElement.interestForElement
---

{{ApiRef("HTML DOM")}}

The **`interestForElement`** property of the {{domxref("HTMLAnchorElement")}} interface gets or sets the interest invoker target element, in cases where the associated {{htmlelement("a")}} element is specified as an interest invoker.

See [Creating an interest invoker](/en-US/docs/Web/API/Popover_API/Using_interest_invokers#creating_an_interest_invoker) for more details.

## Value

> An {{domxref("Element")}} object instance, or `null` if the associated `<a>` element has no interest invoker target.

Reviewer: ```suggestion
An {{domxref("Element")}} object instance, or `null` if the associated `<a>` element does not have a target element set.
```

## Examples

### Basic `interestForElement` usage

> In this example, we use an `<a>` element's `interestForElement` property to set its interest invoker target element and then retrieve its target element's `tagName`. The `tagName` is then printed into the `<a>` element's text content.

Reviewer: ```suggestion
In this example, we use an `<a>` element's `interestForElement` property to set its target element and then retrieve that element's `tagName`. The `tagName` is then printed in the `<a>` element's text content.
```
